os/include/sys/syscall.h

```diff
@@ -99,19 +99,28 @@
 #define SYS_sched_yield                (CONFIG_SYS_RESERVED + 12)
 #define SYS_set_errno                  (CONFIG_SYS_RESERVED + 13)
 
+#ifdef CONFIG_SMP
+#define SYS_sched_getaffinity	       (CONFIG_SYS_RESERVED + 14)
+#define SYS_sched_setaffinity	       (CONFIG_SYS_RESERVED + 15)
+#define SYS_sched_getcpu	       (CONFIG_SYS_RESERVED + 16)
+#define __SYS_sched_smp_end	       (CONFIG_SYS_RESERVED + 17)
+#else
+#define __SYS_sched_smp_end	       (CONFIG_SYS_RESERVED + 14)
```

Comment:
Much minor comment.

All other `__SYS_xxx` represents start of next items.
For example,
```
#ifdef CONFIG_PRIORITY_INHERITANCE
#define SYS_sem_setprotocol            (__SYS_sched_smp_end + 5)
#define __SYS_named_sem                (__SYS_sched_smp_end + 6)
#else
#define __SYS_named_sem                (__SYS_sched_smp_end + 5)
#endif
```
`__SYS_named_sem` items are located next of it.
But only here, it shows end of previous item topic.
How about `#define __SYS_sem` instead of `__SYS_sched_smp_end`?